testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron/EjbElytronDomainSetup.java

```diff
@@ -211,27 +218,16 @@ public void tearDown(final ManagementClient managementClient, final String conta
         System.out.println("tearing down...");
 
         List<ModelNode> updates = new LinkedList<>();
-        updates.add(createRemoveIgnoring(undertowDomainAddress));
-        updates.add(createRemoveIgnoring(httpAuthenticationAddress));
+        updates.add(Util.createRemoveOperation(undertowDomainAddress));
+        updates.add(Util.createRemoveOperation(httpAuthenticationAddress));
         updates.add(Util.getWriteAttributeOperation(ejbRemoteAddress, "connector-ref", "http-remoting-connector"));
-        updates.add(createRemoveIgnoring(ejbDomainAddress));
-        updates.add(createRemoveIgnoring(remotingConnectorAddress));
-        updates.add(createRemoveIgnoring(saslAuthenticationAddress));
-        updates.add(createRemoveIgnoring(domainAddress));
-        updates.add(createRemoveIgnoring(realmAddress));
-
-        try {
-            applyUpdates(managementClient.getControllerClient(), updates, true);
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
-    }
+        updates.add(Util.createRemoveOperation(ejbDomainAddress));
+        updates.add(Util.createRemoveOperation(remotingConnectorAddress));
+        updates.add(Util.createRemoveOperation(saslAuthenticationAddress));
+        updates.add(Util.createRemoveOperation(domainAddress));
+        updates.add(Util.createRemoveOperation(realmAddress));
 
-    private static ModelNode createRemoveIgnoring(PathAddress address) {
-        ModelNode remove = Util.createRemoveOperation(address);
-        // Don't rollback when the AS detects the war needs the module
-        remove.get(OPERATION_HEADERS, ROLLBACK_ON_RUNTIME_FAILURE).set(false);
-        return remove;
+        applyUpdates(managementClient.getControllerClient(), updates, false);
```

Comment:
Note that on the previous L224, the allowFailure param was 'true', now it's false. This change is the primary point of this PR; the rest is just various cleanup. 

The 'createRemoveIgnoring' was pointless method, as headers are not processed for steps in a composite. They are only applicable to the top level operation. And adding the header is likely wrong anyway; we shouldn't be executing ops that fail.